### PR TITLE
PORTALS-2151

### PR DESCRIPTION
**Pull request by jay-hodgson**
Depends on https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/1471, and a new release of SRC

### CODE REVIEW

**Review by yjyoo773 — APPROVED** [review]
Lgtm

File: src/configurations/adknowledgeportal/synapseConfigs/studies.ts [hunk 1]
```diff
@@ -101,8 +102,19 @@ export const studiesDetailsPageProps: DetailsPageProps = {
         {
           name: 'Markdown',
           title: 'Acknowledgement',
-          columnName: 'Acknowledgement',
-          props: {},
+          standalone: true,
```
Comment (jay-hodgson) [inline src/configurations/adknowledgeportal/synapseConfigs/studies.ts:105]: Always show this wiki text

File: src/configurations/adknowledgeportal/synapseConfigs/studies.ts [hunk 2]
```diff
@@ -101,8 +102,19 @@ export const studiesDetailsPageProps: DetailsPageProps = {
...
+          props: {
+            ownerId:'syn12666371',
+            wikiId:'617506'
+          },
+        },
+        {
+          name: 'MarkdownCollapse',
+          columnName: 'ackContext',
```
Comment (jay-hodgson) [inline src/configurations/adknowledgeportal/synapseConfigs/studies.ts:113]: And show the wiki referenced in ackContext in a MarkdownCollapse section.  Include the Copy link, and change the default text description

File: src/portal-components/DetailsPage.tsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ import { BarLoader } from 'react-spinners'
 import { LockedFacet } from 'synapse-react-client/dist/containers/QueryWrapper'
 import DetailsPageTabs from './DetailsPageTabs'
 import { SynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
-import { useGetEntityHeaders } from 'synapse-react-client/dist/utils/hooks/SynapseAPI/useGetEntityHeaders'
+import { useGetEntityHeaders } from 'synapse-react-client/dist/utils/hooks/SynapseAPI/entity/useGetEntityHeaders'
```
Comment (jay-hodgson) [inline src/portal-components/DetailsPage.tsx:30]: This was moved in 3.0.4

**Pull request merged**